lib/register.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:InstiComplaints/modals.dart';
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+class BackgroundMaker extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
```

Comment:
We have stacked two widgets one over the other where the bottom widget is our page and the top widget is the desired shape of the topbar.